deploy/olm-catalog/test/7.12.0-1/manifests/kieapp.crd.yaml

```diff
@@ -1291,158 +1291,421 @@ spec:
+                                    namespace
+                                  properties:
+                                    key:
+                                      description: The key of the secret to select
+                                        from.  Must be a valid secret key.
+                                      type: string
+                                    name:
+                                      description: 'Name of the referent. More info:
+                                        https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names
+                                        TODO: Add other useful fields. apiVersion,
+                                        kind, uid?'
+                                      type: string
+                                    optional:
+                                      description: Specify whether the Secret or its
+                                        key must be defined
+                                      type: boolean
+                                  required:
+                                  - key
+                                  type: object
+                              type: object
+                          required:
+                          - name
+                          type: object
+                        type: array
+                      extraClassPath:
+                        description: ExtraClassPath Allows to add extra jars to the
+                          application classpath separated by colon. Needs to be mounted
+                          on the image before.
+                        type: string
                       image:
-                        description: The image to use for Process Instance Migration
-                          e.g. rhpam-process-migration-rhel8, this param is optional
-                          for custom image.
+                        description: Image The image to use e.g. rhpam-<app>-rhel8,
+                          this param is optional for custom image.
                         type: string
                       imageContext:
-                        description: The image context to use for Process Instance
-                          Migration  e.g. rhpam-7, this param is optional for custom
-                          image.
+                        description: ImageContext The image context to use  e.g. rhpam-7,
+                          this param is optional for custom image.
                         type: string
                       imageTag:
-                        description: The image tag to use for Process Instance Migration
-                          e.g. 7.9.0, this param is optional for custom image.
+                        description: ImageTag The image tag to use e.g. 7.9.0, this
+                          param is optional for custom image.
                         type: string
-                      routeHostname:
+                      imageURL:
```

Comment:
needs a descrip

Reply:
I added it by accident, this one shouldn't be exposed on the crd.